src/main/java/medicaldoctor/controllers/AdditionalLoginController.java

```diff
@@ -0,0 +1,60 @@
+package medicaldoctor.controllers;
+
+import java.net.URL;
+import java.util.ResourceBundle;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.PasswordField;
+import medicaldoctor.backend.LoginService;
+import medicaldoctor.backend.data.LoginResult;
+import medicaldoctor.core.AppSession;
+import medicaldoctor.entities.UserType;
+
+/**
+ *
+ * @author Salmanhussain
+ */
+public class AdditionalLoginController implements Initializable, ParentController{
+    ControllerManager cm;
+    
+    @FXML
+    private Button loginButton;
+    @FXML
+    private PasswordField password;
+    @FXML
+    private Label i_label;
+    
+    @FXML
+    void loginButtonClicked(ActionEvent event) throws Exception{
+        LoginResult result = LoginService.checkAdditionalPassword(password.getText());
+        if(result == LoginResult.SUCCESS){
+            if(AppSession.getActiveUser().getType().equals(UserType.ADMIN)){
```

Comment:
i think we discussed putting the dashboard name as a property of the user type.
then we don't have all these if statements, plus if you add a usertype, you wouldn't have to change any code.